.github/workflows/ci.yml

```diff
@@ -48,4 +62,17 @@ jobs:
           ADAPTER: postgresql
       - name: Run RuboCop
         run: bundle exec rubocop .
-
+      - name: install mysql client
```

Comment:
speaking out of ignorance, why is the client needed?

Reply:
I removed it. When I was figuring out how to make it work, I found a couple of articles saying that you need to install MySQL client to use it on CI.
Looks like it was fixed, so I removed it